ql/src/java/org/apache/hadoop/hive/ql/util/HiveStrictManagedMigration.java

```diff
@@ -1450,7 +1446,9 @@ private TxnCtx generateTxnCtxForAlter(
           result = new TxnCtx(writeId, validWriteIds, txnId);
         } finally {
           if (result == null) {
-            msc.abortTxns(Lists.newArrayList(txnId));
+            AbortTxnsRequest abortTxnsRequest = new AbortTxnsRequest(Lists.newArrayList(txnId));
+            abortTxnsRequest.setErrorCode(TxnErrorMsg.ABORT_MIGRATION_TXN.getErrorCode());
+            msc.abortTxns(abortTxnsRequest);
```

Comment:
there is a method to abort a single txn, why can't you use it?

Reply:
This was already using abortTxns call before. However, changed it now. Done.